manifests/cf-manifest/spec/manifest/jobs_spec.rb

```diff
@@ -37,6 +44,13 @@ def get_job(job_name)
     end
   end
 
+  describe "in order to enforce metron dependency on ingestor" do
```

Comment:
how about "in order to ensure high availability of ingestor"

because this doesn't enforce any dependencies in any way